nuxeo-webengine/nuxeo-webengine-core/src/main/java/org/nuxeo/ecm/webengine/app/JsonWebengineWriter.java

```diff
@@ -95,7 +95,7 @@ protected static void writeException(JsonGenerator jg, Throwable t, MediaType me
         jg.writeNumberField("status", statusCode);
         jg.writeStringField("message", t.getMessage());
         if (jsonFactoryManager.isStackDisplay()
-                || MediaType.valueOf(MediaType.APPLICATION_JSON + "+nxentity").equals(mediaType)) {
+                || MediaType.valueOf(MediaType.APPLICATION_JSON + "+nxentity").equals(mediaType) || true) {
```

Comment:
`|| true` wanted? Or was this test code?

Reply:
Indeed, it's for test.
FI: I had issues in test with jetty and getting the error was really difficult.